modules/gitops-argo-rolouts-cli-ovr.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+:_mod-docs-content-type: CONCEPT
+[id="gitops-argo-rollouts-cli-overview_{context}"]
+= Argo Rollouts CLI Overview
+
+The Argo Rollouts CLI plugin is not required to use Argo Rollouts resources, because you can instead deploy and monitor a Rollout resource using the {OCP} Console or the `oc` command. However, usage of the Argo Rollouts CLI plugin allows you to handle Argo Rollouts resources and display it directly from the command line interface.
+
+The Argo Rollouts CLI plugin interacts with the Argo Rollouts controller and Argo Rollouts resources. The Argo Rollouts CLI plugin enables you to make changes to an Argo Rollouts image, monitor the progress of an Argo Rollouts promotion, proceed with the promotion steps in a canary deployment, terminate a failed Argo Rollouts deployment, and so on.
+
+The Argo Rollouts CLI plugin is an `oc/kubectl` plugin, and integrates directly with the `oc/kubectl` commands.
```

Comment:
`oc/kubectl`: Is this true for downstream as well? Or is it only `oc` commands for downstream?

Reply:
Let me check with the SME.

Reply:
I discussed it a bit more with @eromanova97 and I am keeping the term as "oc kubetcl".